src/rotation.zig

```diff
@@ -328,6 +328,10 @@ pub const RotationModes = struct {
 		const BranchData = packed struct(u6) {
 			enabledConnections: u6,
 
+			pub fn init(blockData: u16) BranchData {
+				return .{.enabledConnections = @intCast(blockData)};
```

Comment:
Should be `@truncate`, `@intCast` will panic or cause undefined behavior if the blockdata is corrupted (which may happen when changing rotation mode)